tests/apk.txt

```diff
@@ -0,0 +1,135 @@
+#!/bin/sh
+
+# Purpose : Testing script for apk support
+# Author  : Álvaro Mondéjar
+# License : MIT
+
+im alpine:3.12 alpine:3.13 alpine:3.14 alpine:latest
```

Comment:
You can also add image `alpine_bash4`? I have a Dockerfile already: https://github.com/icy/pacapt/blob/ng/tests/Dockerfile.alpine_bash4 (the ci system will build new image if necessary: https://github.com/icy/pacapt/blob/d6f429418de2aa6c6b3eca20afe3f3f9c35498dd/tests/test.sh#L19)